Deployment/Mirantis-Guidance.md

```diff
@@ -0,0 +1,274 @@
+# Continuation of container support in Azure Service Fabric
+
+## Abstract 
+
+This guidance is for customers using Azure Service Fabric Service with "with Container" VM images restricted to Windows containers on Windows. Service Fabric Product team is following up on the [May 22 targeted deprecation communication](https://portal.microsofticm.com/imp/v3/comms/trackingid/NS72-DZ8) detailing the phase out of “with container” VM image. The May 22 communication was a follow up of the September 2021 [announcement](https://techcommunity.microsoft.com/t5/containers/updates-to-the-windows-container-runtime-support/ba-p/2788799) on the end of support for Mirantis Container Runtime (MCR). 
+
+Post 22nd September, the “with container” VM images will not receive any updates or patches and SF team strongly recommends using these images for production workloads. We urge customers to consider one of the scenarios listed below and [reach out to support](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request) for any questions or clarifications. Nonconformance to migration guidance and timelines increases the risk of service disruptions and outages. 
+
+## List of affected Azure OS images
+
+Customer is using Windows Server image 2016 with Containers
+- 2016-Datacenter-with-Containers 
+- 2016-Datacenter-with-Containers-g2 
+- 2016-Datacenter-with-Containers-GS 
+
+Customer is using Windows Server image 2019 with Containers
+- 2019-Datacenter-Core-with-Containers 
+- 2019-Datacenter-core-with-containers-g2 
+- 2019-Datacenter-Core-with-Containers-smalldisk 
+- 2019-Datacenter-Core-with-Containers-smalldisk-g2 
+- 2019-Datacenter-with-Containers 
+- 2019-Datacenter-with-Containers-g2 
+- 2019-Datacenter-with-Containers-GS 
+- 2019-Datacenter-with-Containers-smalldisk 
+- 2019-Datacenter-with-Containers-smalldisk-g2
+
+## Migration risk decision guide
+
+This guide is designed to help you assess the effort and risk of each migration option. 
+Criteria for successfully running MCR to host container on Azure Service Fabric cluster.
+1. Acquire a Mirantis Container Runtime license.
+2. Service Fabric runtime needs to be on version 9.0 CU2 or greater.
```

Comment:
add exact version, 9.0.1048.9590 or point to https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-versions

Reply:
Thanks, added it.